source/views/settings/screens/overview/login-button.tsx

```diff
@@ -11,9 +11,10 @@ type Props = {
 
 export function LoginButton(props: Props): JSX.Element {
 	let {loading, disabled, loggedIn, onPress, label} = props
+	let innerText = loggedIn ? 'Signing out of' : 'Signing in to'
 
 	let message = loading
-		? `Logging in to ${label}…`
+		? `${innerText} ${label}…`
 		: loggedIn
 		? `Sign Out of ${label}`
 		: `Sign In to ${label}`
```

Comment:
Name bikeshed: `innerText` -> `loadingText`?

Alternatively:

```suggestion
	let message = loading
		? loggedIn
			? `Signing out of ${label}`
			: `Signing in to ${label}`
		: loggedIn
		? `Sign out of ${label}`
		: `Sign in to ${label}`
```

Or something a bit less terrible like

```ts
	let message
	if (loading) {
		message = loggedIn
		? `Signing out of ${label}`
		: `Signing in to ${label}`
	} else {
		message = loggedIn
		? `Sign out of ${label}`
		: `Sign in to ${label}`
	}
```

Reply:
I probably like the clarity of the last one - the original ternary is on me!

Reply:
Nice. This does add clarity.